src/vm/costs/mod.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2013-2020 Blocstack PBC, a public benefit corporation
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
```

Comment:
Lol are there any other instances of this?  If so can you just fix them in this PR?

Reply:
Yikes, I must have introduced these in my PR that updated copyright to Stacks foundation 🤦🏽 Happy to send a separate PR to correct them all.

Reply:
Not seeing them elsewhere...

Reply:
Opened https://github.com/blockstack/stacks-blockchain/pull/2172